sklearn/impute.py

```diff
@@ -272,6 +282,12 @@ def fit(self, X, y=None):
                                                self.missing_values,
                                                fill_value)
 
+        if self.add_indicator:
+            self.indicator_ = MissingIndicator(
```

Comment:
Let's make this private, i.e. `_indicator`. If we make it public it needs to be described in the Attributes section of the docstring, above.

Reply:
Maybe, let's add the description for the `_indicator` in the Attributes section. It's still can be useful to see `features_` of the MissingIndicator (`imputer.indicator_.features_`).

Reply:
> Maybe, let's add the description for the indicator_ in the Attributes section

Okay